tests/DfE.FindInformationAcademiesTrusts.UnitTests/Services/DataSourceServiceTests.cs

```diff
@@ -3,14 +3,17 @@
 using DfE.FindInformationAcademiesTrusts.Data.Repositories.DataSource;
 using DfE.FindInformationAcademiesTrusts.Services.DataSource;
 using DfE.FindInformationAcademiesTrusts.UnitTests.Mocks;
+using Moq;
```

Comment:
Also...this shows that we must still be referencing the Moq nuget package in one of the csprojs because compilation would fail otherwise